test/models/user_test.rb

```diff
@@ -3,7 +3,13 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'name_or_email returns the name' do
+    user = FactoryBot.create(:user, name: 'ken')
+    assert_equal 'ken', user.name_or_email
+  end
+
+  test 'name_or_email returns the email' do
+    user = FactoryBot.create(:user)
```

Comment:
```suggestion
    user = FactoryBot.create(:user, email: 'ken@example.com', name: '')
```

のようにして、ファクトリの定義をわざわざ見に行かなくても、「なぜこの結果になるのか」がわかるようにしておくとテストの可読性が向上します。

Reply:
変更しました！